src/network.c

```diff
@@ -1426,12 +1433,7 @@ static int parse_packet (sockent_t *se, /* {{{ */
 			status = parse_part_encr_aes256 (se,
 					&buffer, &buffer_size, flags);
 			if (status != 0)
-			{
-				ERROR ("network plugin: Decrypting AES256 "
-						"part failed "
-						"with status %i.", status);
```

Comment:
I was wondering about NOTICE vs ERROR as well. I don't have a strong opinion but I think using ERROR for all messages makes sense, since they are related to the ERROR message at the higher level (the status=-1 message).

For the more complicated case you mention, I think that's a separate issue. To some extent, it's working as expected but there are two cases we might want to avoid: (1) network packages arrive out of order; (2) there's a transient problem and collectd spams logs for that period of time. Anyway, we should discuss that separately.